src/components/Deletebook.js

```diff
@@ -0,0 +1,13 @@
+import { useDispatch } from 'react-redux';
+import { removeBook } from '../redux/books/booksSlice';
+
+function Deletebook() {
+  const dispatch = useDispatch();
+  return (
+    <div>
+      <button type="button" onClick={() => dispatch(removeBook())}>Delete </button>
+    </div>
+  );
```

Comment:
- [ ] Remove book functionality is not working, Please make sure to pass the book id in dispatch, and on the delete button, the book should be removed from the screen.